tools/sloop.c

```diff
@@ -0,0 +1,84 @@
+/**
+ * Library of functions pertaining to Sloop shadows. Slooping is a way
+ * to simulate the built-in snooping functionality of EotL.
+ *
+ * @author gon
+ * 2017-05
```

Comment:
The great thing about using git is it will keep track of all the exact dates and times our files are modified. The date comment here might be nice for people on mud that don't want to go browse to your git repo, but generally it's unnecessary. Also it should go above the @author tag if you're going to include it. Nothing should be below the @tag portion.

Note: Even though dates are kinda superfluous, there is a @version tag that comes in handy for keeping track of major revisions that aren't at the granularity of individual commits.